Jenkinsfile

```diff
@@ -157,6 +157,16 @@ stage('Sanity Check') {
           script: "${docker_run} ${ci_lint}  ./tests/scripts/task_lint.sh",
           label: "Run lint",
         )
+        skip_ci = sh (
+          returnStatus: true,
+          script: './tests/scripts/git_skip_ci.sh',
+          label: "Check if CI should be skipped",
+        )
+        // if (skip_ci == 1) {
```

Comment:
Should these comments be removed as they are duplicated below?

Reply:
sorry this was some work-in-progress code, I should have marked this as a draft since I'm still working on implementing the PR labeling aspect